test/verify/check-realms

```diff
@@ -23,6 +23,34 @@ from testlib import *
 
 import os
 import re
+import subprocess
+
+WAIT_KRB_SCRIPT = """
+# HACK: This needs to work, but may take a minute
+for x in $(seq 1 60); do
+    if getent passwd admin@cockpit.lan; then
+        break
+    fi
+    if systemctl --quiet is-failed sssd.service; then
+        systemctl status --lines=100 sssd.service >&2
+        exit 1
+    fi
+    sleep $x
+done
+
+# This directory should be owned by the domain user
+getent passwd admin@cockpit.lan
```

Comment:
Is this necessary here?  The waiting loop has already made this call.

Reply:
The loop above doesn't have a handling for timeout, i. e. if it completely runs through, the getent passwd call has failed 60 times without aborting. So this will then fail as well and thus fail the test. (Note that this is just moved code; we can rework it if desired, but maybe in a separate PR).

Reply:
Ah, thanks.  I didn't consider the case when it is broken for good.